cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -1099,7 +1094,6 @@ SwiftGenerator::typeToString(
     //
     string currentModule = getSwiftModule(getTopLevelModule(toplevel));
     BuiltinPtr builtin = dynamic_pointer_cast<Builtin>(type);
-    bool nonnull = find(metadata.begin(), metadata.end(), "swift:nonnull") != metadata.end();
```

Comment:
According to the docs, `nonnull` was only for local operations:
https://doc.zeroc.com/ice/3.7/the-slice-language/slice-metadata-directives#id-.SliceMetadataDirectivesv3.7-swift:nonnull